validate-date.js

```diff
@@ -42,6 +42,12 @@ function dateValidator(dateValue, responses, dateFormat) {
         dateFormat = dateValue.includes("-") ? "yyyy-mm-dd" : "mm/dd/yyyy";
         }
 
+        // Check if date separator is different from the dateFormat
```

Comment:
Firstly, thanks for attempting to contribute and raise the quality.

This can be simplified by moving the format check to the beginning of the if block.

`if (!dateValue.includes("-") || !dateValue.includes("/"))`
`    return responses[0];`

Reply:
Or better just add the check [here ](https://github.com/anveshkolluri93/validate-date/blob/master/validate-date.js#L3)

Also, adding a test case to validate this would be great?